src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,100 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private final Pair<K, V>[] storage = new Pair[10];
+    private int count = 0;
```

Comment:
```suggestion
    private int size;
```

it will be initialized by 0 by default